cli/src/installers/dependencyVersionMap.ts

```diff
@@ -26,5 +23,14 @@ export const dependencyVersionMap = {
   "@trpc/next": "^10.26.0",
   "@tanstack/react-query": "^4.29.7",
   superjson: "1.12.2",
+
+  // Prettier
+  prettier: "^2.8.8",
+  "prettier-plugin-tailwindcss": "^0.2.8",
+  "@types/prettier": "^2.7.2",
+  "@ianvs/prettier-plugin-sort-imports": "^3.7.2",
+
+  // Eslint
+  "eslint-config-next": "^13.4.2",
```

Comment:
so this part doesn't make sense to me. why is `eslint-config-next` not in the default but a more opinionated option, strict ts linting, is?

Reply:
Think i forgot it in both configs. Imo we should have everything that is in the more strict version at the moment in the base version, so i dont really know what we want in the more strict version, those hooks rules you mentioned maybe?